node/parachain/src/command.rs

```diff
@@ -268,12 +269,55 @@ pub fn run() -> Result<()> {
 		None => {
 			let runner = cli.create_runner(&*cli.run)?;
 			let collator = cli.run.base.validator || cli.collator;
-			let author_id: Option<H160> = cli.run.author_id;
-			if collator {
-				if author_id.is_none() {
-					return Err("Collator nodes must specify an author account id".into());
-				}
+
+			// Supply the correct author id for wellknown validators.
+			// This isn't super elegant, but the alternative is modifying Substrate
+			// and this will go away when we start signing blocks
+			let author_id = if cli.run.base.shared_params.dev {
+				//TODO what do we actually expect `--dev` to do in the parachain context
+				let alice_public = ecdsa::Pair::from_string("//Alice", None)
+					.expect("Alice is a valid phrase")
+					.public();
+				Some(EthereumSigner::from(alice_public).into_account())
+			} else if cli.run.base.alice {
+				let alice_public = ecdsa::Pair::from_string("//Alice", None)
+					.expect("Alice is a valid phrase")
+					.public();
+				Some(EthereumSigner::from(alice_public).into_account())
+			} else if cli.run.base.bob {
+				let alice_public = ecdsa::Pair::from_string("//Bob", None)
+					.expect("Alice is a valid phrase")
```

Comment:
```suggestion
					.expect("Bob is a valid phrase")
```
and the ones below or could make them all `.expect("Name is a valid phrase"`